packages/palette-docs/content/docs/elements/inputs/Select.mdx

```diff
@@ -65,20 +65,3 @@ of user-benefitting product strategy.
     }}
   </Value>
 </Playground>
-
-### Small select
-
-<Playground>
-  <SmallSelect
-    options={[
-      {
-        text: "Price",
-        value: "price",
-      },
-      {
-        text: "Estimate",
-        value: "estimate",
-      },
-    ]}
-  />
-</Playground>
```

Comment:
Is it possible to remove the corresponding `SmallSelect` code? (Need to check apps to see where it's used first, though.)

Reply:
Oh.. I thought I did:( Maybe somehow did not commit.

Reply:
Just as long as it doesn't break any consuming apps 👍- Will want to check out reaction, volt, positron

Reply:
Yep. Reaction pr that you've merged earlier was addressing the change. And that is the only usage I found.